src/pages/blog/airship-update-july-2020.md

```diff
@@ -0,0 +1,138 @@
+  https://www.airshipit.org/blog/airship-blog-series-4-shipyard-an-evolution-of-the-front-door.html)
+* [_Airship Blog Series 5 - Drydock and Its Relationship to Cluster API_](
+  https://www.airshipit.org/blog/airship-blog-series-5-drydock-and-its-relationship-to-cluster-api.html)
+* [_Airship Blog Series 6 - Armada Growing Pains_](
+  https://www.airshipit.org/blog/airship-blog-series-6-armada-growing-pains.html)
+
+<br>
+
+Stay tuned for more exciting news on the Airship blog series as Airship 2.0 progresses through the beta milestone and
+beyond!
+
+<br>
+
+## **AIRSHIP PROJECT UPDATE**
+
+Each quarter the OpenStack Foundation hosts a community meeting where the latest news about OpenStack and top-level
+projects can be shared. On 25 June, Airship presented a project update alongside OpenStack, Kata Containers, StarlingX,
+Zuul, and pilot project OpenInfra Labs.
+
+Catch up on the community meeting with these resources:
+* [_Recording_](https://zoom.us/rec/share/vJF_FqPgxGJJQ9bntR7vaqM7N7i_X6a81yQa8vtcxU06amK9pV9imWJnfHRSUcQ6),
+  Password: 7W!T*i74
+* [_Slides_](https://docs.google.com/presentation/d/16V82OIYfthb3fFlVoes9jZGKMgDIJZ55F8fXqd1M1hU/edit?usp=sharing)
+* [_Agenda_](https://etherpad.opendev.org/p/OSF_Community_Meeting_Q2)
+
+<br>
+
+The project update included:
+
+* The Alpha Milestone, [_completed in May_](https://www.airshipit.org/blog/airship-update-may-2020/)
+* Results of the [_Virtual PTG in June_](https://etherpad.opendev.org/p/airship-virtual-ptg-2020)
+* The newly introduced [_Vulnerability Management Process_](https://docs.airshipit.org/learn/vulnerabilities.html)
+* An update on GitHub Issues utilization
+* Notification of the Technical Committee (TC) and Working Committee (WC) elections
+* Announcement of the [_Airship User Survery_](https://www.surveymonkey.com/r/YKZ9NC2)
+
+<br>
+
+## **WORKING COMMITTEE ELECTION**
+
+The Technical Committee (TC) is excited to announce the 2020-2021 Working Committee (WC) election! This election will
+follow the same two-week cycle that the Technical Committee (TC) election followed last month.
+
+* Nomination Period: 20-26 July
+* Voting Period: 27 July-02 August
+
+<br>
+
+Anyone who has had changes merged within the last 12 months to an Airship project is eligible to run for the Working
+Committee. Anyone who is a Core Reviewer of any Airship Project before the election will be eligible to vote for the WC
+candidates. There are no term limits, but any organization can fill no more than two of the five seats to encourage
+diversity. The Working Committee will meet regularly in an open forum with times and locations published in community
+channels.
+
+For more details concerning eligibility, responsibilities, and election processes, please consult the
+[_Airship Governance_](https://opendev.org/airship/governance).
+
+<br>
+
+## **AIRSHIP USER SURVEY**
+
+If you are evaluating or running Airship, share your feedback in the [_Airship User Survey_](
```

Comment:
The content in this section is really good, but can we make it more persuasive?

Are you evaluating Airship or using Airship in production?  We want to learn from your experience! Take the Airship User Survey; each response is evaluated by our technical committee and helps us to grow and mature Airship. We will present the 2020 survey results at the Fall Summit in November.

Reply:
Done, used a variation of this - major difference is mentioning the WC instead of the TC to keep consistent with the messaging I received from Ashlee and how OSF does their own user surveys.